includes/class-wc-payments.php

```diff
@@ -510,6 +511,8 @@ public static function init() {
 		include_once __DIR__ . '/compat/multi-currency/class-wc-payments-currency-manager.php';
 		include_once __DIR__ . '/class-duplicates-detection-service.php';
 
+		wcpay_get_container()->get( \WCPay\Internal\LoggerContext::class )->init();
```

Comment:
Is there a specific reason we need to use DI here to run the `init` function? Would a simple `new LoggerContext()` achieve the same result, or is there a need to ensure it's a singleton due to request scope or other considerations?

Reply:
I've made changes to the method and renamed it to `init_hooks` as it is the only function that it carries. I hope this answers your concerns.